src/m365/aad/commands/o365group/o365group-add.ts

```diff
@@ -179,6 +138,95 @@ class AadO365GroupAddCommand extends GraphCommand {
       }, (rawRes: any): void => this.handleRejectedODataJsonPromise(rawRes, logger, cb));
   }
 
+
+  private validateOwners(logger: Logger, args: CommandArgs): Promise<any> {
```

Comment:
Since validating owners and members uses the same logic, let's combine them into one method to avoid duplicating the code. The method would accept an extra argument which is the string of users and which we could call twice, once passing members and once passing owners.